### ISSUE TITLE
Please fix up docs/runtimes/nodejs

### ISSUE THREAD

**Issue by JanKok** [Microsoft/vscode-docs#187]
The instructions on this page are hard to follow for a newcomer like me.

The link https://nodejs.org/download/ is not helpful: what files should I download? How should I install them?

The line "npm install -g express-generator" needs a sudo in front of it to work properly. (I've submitted a pull request for that change.)

The line "express myExpressApp" results in the error:
/usr/bin/env: node: No such file or directory

I tried doing "sudo apt-get install node". Now, "express myExpressApp" gives no error, but gives no joy either. It doesn't create a myExpressApp folder.

Please fix the doc so one can follow the instructions on a PC with freshly installed Ubuntu 14.04 LTS. Thanks!

**Comment by JanKok** [comment]
I did "sudo apt-get install nodejs-legacy" which removed the previously installed "node" package and installed other stuff. Now, "express myExpressApp" works! So, **perhaps the link to https://nodejs.org/download/ can be replaced with an instruction to "sudo apt-get install nodejs-legacy"** (plus whatever the instructions are on other popular distros such as Red Hat or SuSE).

I'm puzzled about some things:

Why are you using a "legacy" package? That makes it sounds like it may not be well-supported in the future.

Why did npm work before I installed node or nodejs-legacy? Was it part of the original Ubuntu installation, or was it installed automatically along with VS Code?

**Comment by gregvanl** [comment]
I've fixed the very unfriendly Node.js download link.  It may be a little while until the fix goes live on code.visualstudio.com.
I'm not a Linux person so I need to discuss some of your other suggestions with colleagues.  For example, should we prefix all npm commands with sudo or include both a Windows and Linux version or are Linux users in the habit of prepending sudo to run install commands as superuser?
Keep your feedback coming, we really appreciate it!

**Comment by Tyriar** [comment]
@gregvanl this is the link I recommend referring people for installing correctly on Linux https://nodejs.org/en/download/package-manager/

**Comment by gregvanl** [comment]
This was added to the Linux setup page long ago.
